apis/placement/v1alpha1/stagedupdate_types.go

```diff
@@ -17,6 +17,9 @@ import (
 // +kubebuilder:subresource:status
 // +kubebuilder:resource:scope=Cluster,categories={fleet,fleet-placement},shortName=crsur
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:printcolumn:JSONPath=`.status.conditions[?(@.type=="Initialized")].status`,name="Initialized",type=string
```

Comment:
discussed offline
* considering print some important info of spec
* considering adding more info (less important) in the -o wide

Reply:
added